### ISSUE TITLE
Police vans

### ISSUE THREAD

**Issue by ouchity** [gamesphere/redwood#32]
Police vans would be good in the future if the arrest system is reworked so people who roleplay can be transported to the police station

**Comment by HamsterJackkingham** [comment]
Agreed also I think police vans look good and I’d use them more then cars so I agree
